src/Content/Version.php

```diff
@@ -33,39 +35,46 @@ public function content(string $language = 'default'): Content
 	{
 		return new Content(
 			parent: $this->model,
-			data:   $this->model->storage()->read($this->id, $language),
+			data:   $this->model->storage()->read($this->id, $this->language($language)),
 		);
 	}
 
+	/**
+	 * Provides simplified access to the absolute content file path.
+	 * This should stay an internal method and be removed as soon as
+	 * the dependency on file storage methods is resolved more clearly.
+	 *
+	 * @internal
+	 */
+	public function contentFile(string $language = 'default'): string
+	{
+		return $this->model->storage()->contentFile($this->id, $this->language($language));
+	}
+
 	/**
 	 * Creates a new version for the given language
+	 *
```

Comment:
For reference later:

```suggestion
	 * @todo Convert to a static method that creates the version initially with all relevant languages
	 *
```